README.md

```diff
@@ -308,6 +308,24 @@ The telepresence background processes `daemon` and `connector` both produces log
 - macOS `~/Library/Logs/telepresence`
 - Linux `~/.cache/telepresence/logs`
 
+## How it works
+
+When Telepresence 2 connects to a Kubernetes cluster, it
+
+ 1. Ensures Traffic Manager is installed in the cluster.
+ 2. Looks for the relevant subnets in the kubernetes cluster.
+ 3. Creates a virtual network device called a _TUN device_ and names it `tel0`.
```

Comment:
The device is named `tun0` on macOS (or `tun1`, `tun2`, etc. whichever is available) so using "tel0" in the text can be confusing for a Mac user.